modules/registry-configuring-storage-azure-user-infra.adoc

```diff
@@ -31,8 +32,13 @@ $ oc edit configs.imageregistry.operator.openshift.io/cluster
 .Example configuration
```

Comment:
Consider removing the block title to remove the following warning:


 31:1  warning  Block titles can only be        AsciiDocDITA.BlockTitle       
                assigned to examples, figures,                                
                and tables in DITA.